### ISSUE TITLE
Support for multi application load balancer on Load Balanced Web Service

### ISSUE THREAD

**Issue by TakaraShinya** [aws/copilot-cli#2604]
I have a lot of sites and rules in a(one) copilot app.
so, our app has an application load balancer, and the maximum number of rules has been reached. 😢 

but we couldn't solve this problem by creating other env and creating new app on same repository. 😭 

if i created other env specified on same vpc, this command(copilot) couldn't create service discovery. because same name. so it shows error msg.

if i created new app on same repository, this command(copilot) coudn't.
it show error msg below. 😓 

```
$ copilot app init cop-wp-second-app
✘ Workspace is already registered with application copilot-wp instead of cop-wp-second-app.
If you'd like to delete the application locally, you can remove the copilot directory.
If you'd like to delete the application and all of its resources, run `copilot app delete`.
✘ workspace already registered with copilot-wp
```

please give me some solution. 🙇

**Comment by efekarakus** [comment]
Hi @TakaraShinya !

Apologies for the inconvenience 😭 

> if i created other env specified on same vpc, this command(copilot) couldn't create service discovery. because same name. so it shows error msg.

We just fixed this yesterday! https://github.com/aws/copilot-cli/pull/2515 and it will go out next release! If you'd like you can build the binary from mainline and use the CLI and it should work!

> if i created new app on same repository, this command(copilot) coudn't.

Another alternative might be to request a quota increase from AWS: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-limits.html 💭 ! That should allow you to increase the number of rules or listeners for the account and I think unblock you

**Comment by TakaraShinya** [comment]
Hi @efekarakus ！
Thank you for your response. 🙇 

I had requested a quota increase the number of rules, and it solved.
But, if our sites more increase, maybe when it will come we couldn't add some new rules.

so, we are looking forward to the next release. 🚀 ‼️

**Comment by Lou1415926** [comment]
The fix is now released in [v1.9.0](https://github.com/aws/copilot-cli/releases/tag/v1.9.0)! 🎊
